test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -44,22 +44,22 @@ Describe "Certificate Provider tests" -Tags "CI" {
     }
 
     Context "Get-Item tests" {
-        it "Should be able to get a certificate store, path: <path>" -TestCases $testLocations {
+        it "Should be able to get a certificate store, path: <path>" -TestCases $currentUserMyLocations {
             param([string] $path)
             $expectedResolvedPath = Resolve-Path -LiteralPath $path
             $result = Get-Item -LiteralPath $path
-            $result | should not be null
+            $result | Should -Not -BeNullOrEmpty
             $result | ForEach-Object {
                 $resolvedPath = Resolve-Path $_.PSPath
-                $resolvedPath.Provider | should be $expectedResolvedPath.Provider
-                $resolvedPath.ProviderPath.TrimStart('\') | should be $expectedResolvedPath.ProviderPath.TrimStart('\')
+                $resolvedPath.Provider | Should -Be $expectedResolvedPath.Provider
+                $resolvedPath.ProviderPath.TrimStart('\') | Should -Be $expectedResolvedPath.ProviderPath.TrimStart('\')
             }
         }
         it "Should return two items at the root of the provider" {
-            (Get-Item -Path cert:\*).Count | should be 2
+            (Get-Item -Path cert:\*).Count | Should -Be 2
```

Comment:
Please re-look the PR - we should use new syntax `Should -HaveCount`.

https://github.com/pester/Pester/blob/6cbac753e9e382aa3b1c482a3424c99602c01960/en-US/about_Should.help.txt